I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 355, in script
    Dorfbewohner "AAAAHHH! Hilfe!" #geschrei vom dorfeingang
Exception: Sayer 'Dorfbewohner' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 355, in script
    Dorfbewohner "AAAAHHH! Hilfe!" #geschrei vom dorfeingang
  File "D:\Ablage\Studium\Howdey_Abbey\renpy-8.1.3-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "D:\Ablage\Studium\Howdey_Abbey\renpy-8.1.3-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'Dorfbewohner' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
Howdey Abbey 1.0
Fri Jan 26 18:31:48 2024
